banyand/measure/measure_write.go

```diff
@@ -196,8 +195,8 @@ func (w *writeCallback) Rev(message bus.Message) (resp bus.Message) {
 	return
 }
 
-func familyIdentity(name string, flag byte) []byte {
-	return bytes.Join([][]byte{[]byte(name), {flag}}, nil)
+func familyIdentity(name string, flag []byte) []byte {
+	return bytes.Join([][]byte{tsdb.Hash([]byte(name)), flag}, nil)
```

Comment:
It seems strange to use `bytes.Join` here?

Reply:
Any thoughts here?  I applied this pattern everywhere.

Reply:
> Any thoughts here? I applied this pattern everywhere. 

more or less OK for me. Let's optimize later~